dts/bindings/mtd/jedec,spi-nor-common.yaml

```diff
@@ -8,13 +8,22 @@
 properties:
   jedec-id:
     type: uint8-array
-    required: true
+    required: false
     description: JEDEC ID as manufacturer ID, memory type, memory density
 
+  sfdp-bfp:
+    type: uint8-array
+    required: false
+    description: |
+      Contains the 32-bit words in little-endian byte order from the
+      JESD216 Serial Flash Discoverable Parameters Basic Flash
+      Parameters table.  This provides flash-specific configuration
+      information in cases were runtime configuration is not desired.
+
   has-be32k:
     type: boolean
     required: false
-    description: Indicates the device supports the BE32K (0xD8) command
+    description: Not used after Zephyr 2.3.0
```

Comment:
I assume that means this property is deprecated and we can remove it in the future?